lib/rules/import-no-redux-combine-reducers.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @fileoverview Disallow combineReducers import from redux
+ * @author Automattic
+ * @copyright 2017 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const ERROR_MESSAGE = 'combineReducers should be imported from state/utils not redux';
+
+module.exports = {
+	meta: {
+		docs: {
+			description: 'Disallow combineReducers import from redux',
+			category: 'Possible Errors'
+		}
+	},
+	create( context ) {
+		return {
+			ImportDeclaration( node ) {
+				if ( node.source.value === 'redux' ) {
+					const combineReducers = node.specifiers.filter(
```

Comment:
[`Array#some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some?v=control) would be more semantically meaningful for the intent here and avoid iterating the entire set if detected early on.